### PR TITLE
boards/mulle: Add more ADC lines to default configuration

### PR DESCRIPTION
**Pull request by gebart**
Added some more pins to the default configuration.

### CODE REVIEW

File: boards/mulle/include/periph_conf.h [hunk 1]
```diff
@@ -142,12 +142,23 @@ extern "C"
  */
 static const adc_conf_t adc_config[] = {
     /* dev, pin, channel */
-    { ADC0, GPIO_UNDEF, 26 },       /* internal: temperature sensor */
-    { ADC0, GPIO_UNDEF, 27 },       /* internal: band gap */
-    { ADC0, GPIO_UNDEF, 29 },       /* internal: V_REFSH */
-    { ADC0, GPIO_UNDEF, 30 },       /* internal: V_REFSL */
-    { ADC1, GPIO_UNDEF,  0 },       /* connected to  Mulle Vbat/2 on PGA1_DP */
-    { ADC1, GPIO_UNDEF, 19 },       /* connected to Mulle Vchr/2 on PGA1_DM  */
+    [ 0] = { ADC1, GPIO_UNDEF, 26 },       /* internal: temperature sensor */
```
Comment (cgundogan) [inline boards/mulle/include/periph_conf.h]: Was it intended to change from `ADC0` to `ADC1` for pins 26,27,29 and 30?

Comment (gebart) [inline boards/mulle/include/periph_conf.h]: :+1: Good catch. It was intentional with the intention that if you don't use any expansion port analog pins you only need to power the ADC0 module in order to measure battery voltage, charger voltage and, measuring the 1 V band gap to compute the AVDD.

**Comment by cgundogan** [comment]
untested ACK

**Comment by cgundogan** [comment]
Murdock approves. GO (Runtime of 11m:21s, wow)

**Pull request merged**